### DISCUSSION TITLE
Ipv6下的DNS解析问题

### DISCUSSION THREAD

**Discussion by zzz6839** [vernesong/OpenClash#2180] [Q&A]
尝试纯IPV6环境下访问淘宝报DNS_PROBE_FINISHED_NXDOMAIN，各类测试加进程显示域名解析不出IP，不知道DNS设置这边哪里出问题了，求助大神

Connection-specific DNS Suffix: lan
Description: Intel(R) Wi-Fi 6E AX210 160MHz
Physical Address: ‎70-1A-B8-97-45-CD
DHCP Enabled: Yes
IPv4 Address: 192.168.1.102
IPv4 Subnet Mask: 255.255.255.0
Lease Obtained: Thursday, March 24, 2022 12:31:12 AM
Lease Expires: Thursday, March 24, 2022 12:31:11 PM
IPv4 Default Gateway: 192.168.1.1
IPv4 DHCP Server: 192.168.1.1
IPv4 DNS Server: 192.168.1.1
IPv4 WINS Server: 
NetBIOS over Tcpip Enabled: Yes
IPv6 Address: 240e:390:803:e2a0::a9f
Lease Obtained: Thursday, March 24, 2022 12:23:00 AM
Lease Expires: Thursday, March 24, 2022 10:45:32 AM, 240e:390:803:e2a0:8567:b821:4f24:77c0
Temporary IPv6 Address: 240e:390:803:e2a0:650a:5873:da5b:7f40
Link-local IPv6 Address: fe80::8567:b821:4f24:77c0%23
IPv6 Default Gateway: fe80::1%23
IPv6 DNS Server: fe80::1%23

`自定义上游DNS服务器 
追加上游DNS 
Fake-IP 持久化 
允许IPv6类型DNS解析 
禁止Dnsmasq缓存DNS 

Nameserver 223.5.5.5/223.6.6.6 UDP
Nameserver [2400:3200::1/2400:3200:baba::1]:53 UDP
Fallback doh.pub/dns-query HTTPS

Adguard 上游DNS 

127.0.0.1:7874
[::1]:7874

无缓存
`

其他ipv6的应该都开了，但是openclash控制面板没有看到局域网ipv6对应进程。

**Comment by zzz6839** [comment]
尝试发现openclash未能接管局域网内电脑设备的ipv6流量，但是NAS可以，这是什么情况？

**Comment by vernesong** [comment]
ipv6跟v4一样的，设备要把网关和DNS改成路由的

**Reply by zzz6839** [reply]
一样的啊，都是路由ULA Prefix，本地链路地址
IPv6 Default Gateway: fe80::1%23
IPv6 DNS Server: fe80::1%23

**Comment by MBRjun** [comment]
Clash日志等级开调试，日志看看
